src/evaluator/mod.rs

```diff
@@ -21,7 +21,7 @@ impl Evaluator {
 
     fn is_truthy(obj: Object) -> bool {
         match obj {
-            Object::Null | Object::Bool(false) => false,
+            Object::Null | Object::Bool(false) | Object::Int(325) => false,
```

Comment:
我觉得应该是以 3.5 分（350）为界去判定：
```suggestion
            Object::Null | Object::Bool(false) => false,
            Object::Int(x) => x > 350,
```